frontend/src/components/App.js

```diff
@@ -134,6 +134,13 @@ export default class App extends Component {
   }
 }
 
-const root = ReactDOM.createRoot(document.getElementById('app'));
+const loadApp = () => {
+  if (systemClient.loading) {
+    setTimeout(loadApp, 200);
+  } else {
+    const root = ReactDOM.createRoot(document.getElementById('app'));
+    root.render(<App />);
+  }
+};
```

Comment:
The cookie data stored on the phone takes a while top be decrypted and loaded, so we wait for a `loading` flag to load the app

Reply:
On web, this flag is always `false`